src/test/e2e/29_config_file_test.go

```diff
@@ -18,17 +18,23 @@ func TestConfigFile(t *testing.T) {
 	e2e.SetupWithCluster(t)
 
 	var (
-		path   = fmt.Sprintf("zarf-package-config-file-%s.tar.zst", e2e.Arch)
-		dir    = "examples/config-file"
-		config = "zarf-config.toml"
+		path       = fmt.Sprintf("zarf-package-config-file-%s.tar.zst", e2e.Arch)
+		dir        = "examples/config-file"
+		config     = "zarf-config.toml"
+		configPath = filepath.Join(dir, config)
 	)
+	// defer unsetting the ZARF_CONFIG so that we can test w/ the env var and w/ the flag both
+	defer os.Unsetenv("ZARF_CONFIG")
 
 	e2e.CleanFiles(path)
 
+	// Test the config file flag
+	os.Unsetenv("ZARF_CONFIG")
```

Comment:
This was originally necessary but can now be deleted. 
IMO each test should expect no environment variables to have been set it other tests as was happening before